### PR TITLE
dev to staging

### PR DESCRIPTION
**Pull request by stolarczyk**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -2,6 +2,24 @@
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) and [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) format. 
 
+## [0.2.1] -- unreleased
```
Comment (nsheff) [inline changelog.md]: I think this is a major update and should go to 0.3.0

File: refgenieserver/_version.py [hunk 2]
```diff
@@ -1 +1 @@
-__version__ = "0.2.0"
+__version__ = "0.2.1-dev"
```
Comment (nsheff) [inline refgenieserver/_version.py]: 0.3.0

File: refgenieserver/const.py [hunk 3]
```diff
@@ -13,8 +13,9 @@
 TEMPLATES_PATH = os.path.join(os.path.dirname(os.path.abspath(__file__)), TEMPLATES_DIRNAME)
 STATIC_DIRNAME = "static"
 STATIC_PATH = os.path.join(os.path.dirname(os.path.abspath(__file__)), STATIC_DIRNAME)
-TGZ = dict(ext=".tgz", flags="-cvzf")
-TAR = dict(ext=".tar", flags="-cvf")
 FILE_SIZE_UNITS = ['B', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB']
 LOG_FORMAT = "%(levelname)s in %(funcName)s: %(message)s"
 MSG_404 = "No such {} on server"
+DESC_PLACEHOLDER = "No description"
+CHECKSUM_PLACEHOLDER = "No checksum"
```
Comment (nsheff) [inline refgenieserver/const.py]: digest vs checksum?

**Comment by nsheff** [comment]
Looks great!

**Review by nsheff — APPROVED** [review]

File: refgenieserver/server_builder.py [hunk 4]
```diff
@@ -1,150 +1,126 @@
...
-    # import tarfile
-    # with tarfile.open(output, "w:gz") as tar:
-    #     tar.add(path, arcname=os.path.basename(path))
     if os.path.exists(path):
-        enclosing_dir = os.path.dirname(path)
-        entity_name = os.path.basename(path)
-        # use -C (cd to the specified dir before taring) option not to include the directory structure in the archive
-        run("tar -C {} {} {} {}".format(enclosing_dir, flags, output, entity_name), shell=True)
+        if gz:
+            cmd = "cd {}; tar -cvf - . | pigz > {}" if is_command_callable("pigz") else "cd {}; tar -cvzf {} ."
```
Comment (nsheff) [inline refgenieserver/server_builder.py:121]: A better approach to solve this: https://github.com/databio/pypiper/blob/67908f2ee5f51fa5fdddb67eb6d7891aefeeda6a/pypiper/ngstk.py#L66-L69

Comment (stolarczyk) [reply refgenieserver/server_builder.py:121]: what's the improvement?

Did you mean that in `ngstk` you determine number of available cores?

Comment (nsheff) [reply refgenieserver/server_builder.py:121]: never mind. I just thought it was  easier to understand what's going on and simpler. just a programming improvement, not a practical improvement. you can set a variable.

**Pull request merged**